app/graphql/types/query_type.rb

```diff
@@ -39,6 +39,17 @@ def organizations(city:, state:)
       Organization.where("city ILIKE ? AND state ILIKE ?", "%#{city}%", "%#{state}%")
     end
 
+    field :organizationsLatLon, [Types::OrganizationType], null: false, description: "Retrieve all organiations within `r` miles of a given lat/lon. The default radius (`r`) is 20 miles." do
+      argument :latitude, Float, description: "A latitude coordinate."
+      argument :longitude, Float, description: "A longitude coordinate." 
+      argument :radius, Integer, required: false, description: "(OPTIONAL) The search radius, in miles, from the provided `latitude` / `longitude`. Default value is 20 miles."
+    end
+
+    def organizationsLatLon(latitude:, longitude:, radius:20)
+      coordinates = [latitude, longitude]
+      Organization.near(coordinates, radius)
+    end
```

Comment:
It would also be possible to keep using the same `organizations` query endpoint we already have, and use a conditional like
```
if latitude && longitude
   Organization.near(coordinates, radius)
else
  Organization.where("city ILIKE ? AND state ILIKE ?", "%#{city}%", "%#{state}%")
end
```

Do we have any strong feelings about making them two different endpoints? My main thought is that FE is going to get back a key called `organizationLatLon` and will have to make a new method to parse it, as opposed to reusing the method they already have to parse the `organizations` endpoint

Reply:
combined into one endpoint